tests/lib/app_keys.py

```diff
@@ -0,0 +1,39 @@
+
+def assert_api_product_in_application(developer_app_keys_api, dev_email, app_name, key_id, api_product_name):
```

Comment:
Not too keen on the term `assert` here but I can't come up with a name for what this method is doing. It's more along the lines of `add_api_product_if_not_exist_in_application`

Reply:
My expectation is that when this function is called it will either return having added the api product to the given application key or it will have thrown an exception. Even though we do not explicitly throw an exception the underlying library call will throw an exception if there is a problem. How about `ensure_api_product_in_application`?

Reply:
`ensure_api_product_in_application` sounds good